tests/e2e/tests/i18n/settings.spec.ts

```diff
@@ -14,103 +16,118 @@ test.describe('Settings', () => {
-      await page.getByRole('checkbox', { name: 'indexable' }).click();
-
-      /**
-       * Try to publish. It should fail because content has a min: 2 constraint
-       * Then it should display the "missing component" validation error (as a button)
-       */
-      await page.getByRole('button', { name: 'Publish' }).click();
-
-      /**
-       * Create the missing components in the "content" dynamic zone.
-       */
-      await page.getByRole('button', { name: 'There are 2 missing components' }).click();
-      await page.getByRole('button', { name: 'Product carousel' }).click();
-      await page.getByRole('button', { name: 'Product carousel' }).click();
-      await page
-        .getByRole('region', { name: /Product carousel/ })
-        .getByRole('textbox', { name: 'title' })
-        .fill('Magliette');
-      await page.getByRole('button', { name: 'There is 1 missing component' }).click();
-      await page.getByRole('button', { name: 'Hero image' }).click();
-      await expect(page.getByText('content (2)')).toBeVisible();
-
-      /**
-       * Successfully publish the entry once the data is valid.
-       */
-      await page.getByRole('button', { name: 'Publish' }).click();
-      await findAndClose(page, 'Success:Published');
+    /**
+     * Next, we'll navigate to our shop single type & add the a localised version of this document.
+     */
+    await page.getByRole('link', { name: 'Content Manager' }).click();
+    await page.getByRole('link', { name: 'Shop' }).click();
+    await page.waitForURL(/\/admin\/content-manager\/single-types\/api::shop.shop(\?.*)?/);
+    await expect(page.getByRole('heading', { name: 'UK Shop' })).toBeVisible();
+    await page.getByRole('combobox', { name: 'Locales' }).click();
+    expect(await page.getByRole('option').all()).toHaveLength(5);
+    for (const locale of [...LOCALES, 'Italian (it)']) {
+      await expect(page.getByRole('option', { name: locale })).toBeVisible();
+    }
+    await page.getByRole('option', { name: 'Italian (it)' }).click();
+    await expect(page.getByRole('heading', { name: 'Shop' })).toBeVisible();
+
+    /**
+     * Because this is technically a new entry, we should assert that the document panel actions are disabled.
+     */
+    await expect(page.getByRole('button', { name: 'Publish' })).toBeDisabled();
+    await expect(page.getByRole('button', { name: 'Save' })).toBeDisabled();
+
+    if (edition === 'EE') {
+      // Assert the menu state for enterprise edition
+      await expect(page.getByRole('button', { name: 'More document actions' })).toBeEnabled();
+      await page.getByRole('button', { name: 'More document actions' }).click();
+      await expect(page.getByRole('menuitem', { name: 'Add to release' })).toBeEnabled();
+      await expect(page.getByRole('menuitem', { name: 'Unpublish' })).toBeDisabled();
+      await expect(page.getByRole('menuitem', { name: 'Discard changes' })).toBeDisabled();
+      await page.keyboard.press('Escape');
```

Comment:
Not closing the menu will cause the test to fail but could do

Reply:
I didn't add the check. I think the fact that it fails is enough, we close the menu like this in several places and since it would make sense to me to do the check everywhere we do this that just feels like added noise. But if you see it differently let me know.